### ISSUE TITLE
Wrong peer dependencies on addons sub-packages

### ISSUE THREAD

**Issue by evanliomain** [storybooks/storybook#2728]
### Issue details

@ storybook/addon-* should peer depend on same version as @ storybook/addons

### Steps to reproduce

`npm install --save-dev @storybook/cli`
`getstorybook --use-npm`
`npm shrinkwrap -dev`

```bash
npm ERR! Linux 3.13.0-43-generic
npm ERR! argv "/home/evan/.config/fnm/bin/node" "/home/evan/.config/fnm/bin/npm" "shrinkwrap" "-dev"
npm ERR! node v6.9.5
npm ERR! npm  v3.10.10

npm ERR! Problems were encountered
npm ERR! Please correct and try again.
npm ERR! peer invalid: @storybook/addons@^3.3.0, required by @storybook/addon-actions@3.4.0-alpha.1
npm ERR! peer invalid: @storybook/addons@^3.3.0, required by @storybook/addon-links@3.4.0-alpha.1
npm ERR! peer invalid: @storybook/addons@^3.3.0, required by @storybook/addon-notes@3.4.0-alpha.1
```

### Please specify which version of Storybook and optionally any affected addons that you're running

- @ storybook/addon-* 3.4.0-alpha.1 and 3.4.0-alpha.2

### Affected platforms 

- node v6.9.5
- npm  v3.10.10

**Comment by Hypnosphi** [comment]
@shilman so lerna doesn't update those authomatically?

**Comment by Hypnosphi** [comment]
I also wonder why getstorybook installs alpha

**Comment by evanliomain** [comment]
@Hypnosphi it was the 3.4.0-alpha.1. Maybe it's related with the "accidentally published 3.4.0-alpha.1 on the latest NPM tag".

**Comment by danielduan** [comment]
Is this fixed with that dependency PR you created? @Hypnosphi?

**Comment by Hypnosphi** [comment]
I don't think the general issue can be fixed without adding `*` as peerDep versions. At least, I'm not aware of other ways to match all the future alpha versions

**Comment by Hypnosphi** [comment]
those peer deps are removed in #3383

**Comment by Hypnosphi** [comment]
Released as [`4.0.0-alpha.3`](https://github.com/storybooks/storybook/releases/tag/v4.0.0-alpha.3)
